docs/3rd_party/pendulum/snippets/weeks_in_interval.py

```diff
@@ -0,0 +1,25 @@
+import pendulum
+
+
+def weeks_in_interval(start_date: str, end_date: str) -> list[pendulum.DateTime]:
+    """
+    Generate a list of weeks within the interval between start_date and end_date
+
+    Args:
+        start_date (str): The start date of the interval.
+        end_date (str): The end date of the interval.
+
+    Returns:
+        list[pendulum.DateTime]: A list of weeks within the interval.
+    """
```

Comment:
```suggestion
    """
    Generate a list of weeks within the interval between start_date and end_date

    Args:
        start_date: The start date of the interval.
        end_date: The end date of the interval.

    Returns:
        A list of weeks within the interval.
    """
```
just staying consistent.